packages/cadl-ranch-expect/src/decorators.ts

```diff
@@ -238,6 +238,29 @@ export function $scenarioService(
     derivedModels: [],
     projectionsByName: [],
   } as any);
-  context.call($server, target, "http://localhost:3000", "TestServer endpoint");
+
+  context.call($server, target, "{endpoint}", "Test server endpoint", {
+    kind: "Model",
+    properties: new Map().set("endpoint", {
+      name: "endpoint",
+      type: {
+        kind: "Scalar",
+        name: "string",
+        derivedScalars: [],
+        projections: [],
+        projectionsByName: [],
+        isFinished: false,
+        decorators: [],
+        namespace: { kind: "Namespace", name: "TypeSpec", namespace: context.program.getGlobalNamespaceType() },
+      },
```

Comment:
I would suggest use the TypeSpec.string directly (instead of a mock one)
```
type: context.program.getGlobalNamespaceType().namespaces.get("TypeSpec").scalars.get("string"),
```